drivers/serial/uart_stm32.c

```diff
@@ -1319,7 +1319,7 @@ static int uart_stm32_async_rx_enable(const struct device *dev,
 	}
 
 	if (data->dma_rx.enabled) {
-		LOG_WRN("RX was already enabled");
+		LOG_ERR("RX was already enabled");
```

Comment:
Is LOG_ERR really required here ?
Also this change is not justified/documented

Reply:
removed